sqs/templates/lambda_policy.json.tpl

```diff
@@ -0,0 +1,18 @@
+{
+  "Version": "2012-10-17",
+  "Id": "sns_topic_policy",
+  "Statement": [
+    {
+      "Sid": "sns_topic_statement",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:${region}:${account_id}:${sqs_name}",
+      "Condition": {
+        "ArnEquals": {
+          "aws:SourceArn": "arn:aws:lambda:${region}:${account_id}:*"
```

Comment:
Can the ARN be made specific to a Lambda, rather than using a wildcard?

Reply:
This isn't needed at all. The access policies are for when something within AWS is triggering the queue like an SNS topic. This policy is only being used on the file-format queue and messages are being sent to that by the download files lambda which has the correct permissions already. I've changed [terraform-environments](https://github.com/nationalarchives/tdr-terraform-environments/pull/93/commits/02d916f687187e798937d6bf15017da1fe634e2e) so that the queue uses the default policy.